cmd/metrics-v2.go

```diff
@@ -89,6 +90,7 @@ func init() {
 		getS3TTFBMetric(),
 		getNotificationMetrics(),
 		getDistLockMetrics(),
+		getIAMNodeMetrics(),
```

Comment:
should be adding to nodeGroups for node endpoint - peerMetricsGroups and clusterMetricsGroups are both collected by clusterCollector ( will result in dups)

Reply:
Also, should metrics published to cluster endpoint have cluster namespace and node name as variable label here?

Reply:
no they are presented on purpose with server labels @poornas